apps/modernization-ui/src/pages/patient/profile/addresses/AddressesTable.tsx

```diff
@@ -129,17 +127,18 @@ export const AddressesTable = ({ patient }: Props) => {
     const [remove] = useDeletePatientAddressMutation();
 
     useEffect(() => {
-        fetch({
-            variables: {
-                patient: patient,
-                page: {
-                    pageNumber: currentPage - 1,
-                    pageSize: TOTAL_TABLE_DATA
-                }
-            },
-            notifyOnNetworkStatusChange: true
-        });
-    }, [currentPage]);
+        patient &&
+            fetch({
+                variables: {
+                    patient: patient?.id,
+                    page: {
+                        pageNumber: currentPage - 1,
+                        pageSize: TOTAL_TABLE_DATA
+                    }
+                },
+                notifyOnNetworkStatusChange: true
+            });
+    }, [currentPage, patient]);
```

Comment:
question: Is `patient` needed as a dependency? It's passed in and not part of the state of this component

Reply:
Yes because patient.ID is being passed into the API